contents/docs/experiments/manual.mdx

```diff
@@ -180,26 +180,26 @@ if variant == 'variant_name' {
 // Feature flags are not supported in our Java SDK.
 // To run an experiment using our Java SDK, 
 // see our docs on running an experiment without feature flags
-// TODO add link
+// https://posthog.com/docs/experiments/running-experiments-without-feature-flags
```

Comment:
I think we should just link this as a note below the `<MultiLanguage>` snippet, saying "For the API, Java, Rust, and Elixir, see our docs on running an experiment without feature flags"